### PR TITLE
Feature/service

### PR DESCRIPTION
**Pull request by gulli301185**

### CODE REVIEW

File: src/components/service/Service.jsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+               вопросы. <br /> Ниже представленны наиболее популярные.
+            </p>
+
+            <Sellect />
+         </SelectContent>
+         <CardApplication />
+         <Footer />
+      </div>
+   )
+}
```
Comment (erkinovakurmanzhan1) [inline src/components/service/Service.jsx:46]: checked

**Review by Abdybakieva — COMMENTED** [review]
checked :)

**Comment by Kamchybekk** [comment]
Screen Shot 2023-04-14 at 18 51 41

**Comment by RussoOrmon** [comment]
checked

File: src/components/UI/select/Sellect.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+   Accordion,
+   AccordionDetails,
+   AccordionSummary,
+   Typography,
+} from '@mui/material'
+import styled from '@emotion/styled'
+import { serviceData } from '../../../utlis/services/service_data'
+import { ReactComponent as ExpandMoreIcon } from '../../../assets/serviceIcons/Указатель.svg'
+
+function Sellect() {
```
[the remark this answers is not in the document]
Comment (songhee24) [reply src/components/UI/select/Sellect.jsx:12]: so you need to provide props which will render data

File: src/components/card/CardApplication.jsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+import { FormLabel, Grid, InputAdornment } from '@mui/material'
+import { styled } from '@mui/material/styles'
+import { ReactComponent as ButtonIcon } from '../../assets/serviceIcons/ButtonIcon.svg'
+import { ReactComponent as Users } from '../../assets/serviceIcons/Users.svg'
+import { ReactComponent as Phone } from '../../assets/serviceIcons/phoneForModal.svg'
+import { ReactComponent as Women } from '../../assets/serviceIcons/Women.svg'
+import Button from '../UI/Button'
+import Input from '../UI/input/Input'
+
+export default function CardApplication() {
```
Comment (songhee24) [inline src/components/card/CardApplication.jsx:14]: also there we need to collect the form data such phone number and pass to parent component with lifting up pattern

**Review by songhee24 — CHANGES_REQUESTED** [review]

File: src/components/card/CardApplication.jsx (hunk 3 above)
Comment (songhee24) [inline src/components/card/CardApplication.jsx:14]: please pay attention too

**Review by songhee24 — CHANGES_REQUESTED** [review]

File: src/components/card/CardApplication.jsx (hunk 3 above)
Comment (gulli301185) [reply src/components/card/CardApplication.jsx]: sorry)

**Review by songhee24 — APPROVED** [review]

**Pull request merged**